spec/katello_certs_check_spec.rb

```diff
@@ -15,4 +15,11 @@ def fixture(filename)
     expect(stdout).to eq ''
     expect(status.exitstatus).to eq 1
   end
+
+  it 'with password on key' do
+    stdout, stderr, status = Open3.capture3(command)
```

Comment:
You probably need to modify the command to add a filename and store the file in git or generate it on the fly.